### ISSUE TITLE
Crash when repack OnePlus ops file

### ISSUE THREAD

**Issue by MlgmXyysd** [bkerler/oppo_decrypt#35]
```batch
python3 opscrypto.py encrypt extract --projid=19805 --firmwarename=19805 --savename=19805.ops

Oneplus CryptTools V1.3 (c) B. Kerler 2019-2021
MIT License
----------------------------
Traceback (most recent call last):
  File "opscrypto.py", line 697, in <module>
    main()
  File "opscrypto.py", line 630, in main
    item, pos = copyitem(item, directory, pos, wf)
  File "opscrypto.py", line 517, in copyitem
    pos += rlen
TypeError: unsupported operand type(s) for +=: 'int' and 'NoneType'
```
I tried to add these code after line 517:
```python
if rlen == None:
            rlen = 0
```
Then ops can generate, but cannot load.
![image](https://user-images.githubusercontent.com/17919858/119284567-9a8a9880-bc72-11eb-8c7c-47769d8f3ecd.png)
(MsmDownloadTool_ops_5.1.92.0_reserve)

**Comment by MlgmXyysd** [comment]
@bkerler

**Comment by bijleeeeeeeee** [comment]
@bkerler - I have the same error while trying to encrypt OnePlus ops file

**Comment by bkerler** [comment]
I will have a look :)

**Comment by bkerler** [comment]
Solved with commit 821e0ee

**Comment by MlgmXyysd** [comment]
works well :)
